### ISSUE TITLE
`WRONG_INDENTATION` false-positive: "the same number of indents to the opening and closing quotes was expected"

### ISSUE THREAD

**Issue by petertrr** [saveourtool/diktat#1490]
## Describe the bug
I believe, caused by https://github.com/saveourtool/diktat/commit/998d0e97bf838eec12f7852568b5d4932401e707

## Observed behavior
https://github.com/saveourtool/save-cli/security/code-scanning/132
https://github.com/saveourtool/save-cli/security/code-scanning/131
Reproducible on diktat-demo with this snippet:
```kotlin
package com.saveourtool.save.reporter.plain

open class PlainTextReporter {
    @Suppress("MAGIC_NUMBER", "MagicNumber")
    override fun afterAll() {
        out.write(
            "$status: $total tests, $passRate%% successful, failed: $failed, skipped: $skipped"
                .encodeToByteArray()
            )
    }
}
```

## Environment information
* diktat version: 1.2.3

**Comment by 0x6675636b796f75676974687562** [comment]
Originally from [saveourtool/save-cli/security/code-scanning/131](https://github.com/saveourtool/save-cli/security/code-scanning/131), caused by 998d0e97 as a part of #1364.

@Arrgentum, could you take a look please?
